sdk/appcenter-codepush/src/androidTest/java/com/microsoft/appcenter/codepush/CodePushAndroidTest.java

```diff
@@ -0,0 +1,20 @@
+package com.microsoft.appcenter.codepush;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class CodePushAndroidTest {
+
+    @Test
+    public void useAppContext() throws Exception {
+        
+        // Context of the app under test.
```

Comment:
All our comments use `/*  */` except the ones generated by Android Studio do disable a warning.

Also every blank line in a method must be followed by a new comment to start the new block.

Reply:
@guperrot Fixed